modules/store-devtools/src/devtools.ts

```diff
@@ -121,13 +133,15 @@ export class StoreDevtools implements Observer<any> {
       value: toSignal(state$, { manualCleanup: true, requireSync: true }),
     });
 
-    this.extensionStartSubscription = extensionStartSubscription;
-    this.stateSubscription = liftedStateSubscription;
```

Comment:
For reference, why not just unsubscribe from these in the `ngOnDestroy` instead of adding a destroy Subject?

Reply:
Well, my VSCode initially marked them as unused, so I removed them first. Then I added `takeUntil` since I saw it's already being used in other places too.

Reply:
I can revert properties back and unsubscribe.

Reply:
I think that approach would be better in this case because the subscription references are already there.

Reply:
Updated